INFO:root:app.py: API-call GetHumidityValues
INFO:root:app.py: API-call GetHumidityValues
INFO:root:app.py: API-call GetHumidityValues
INFO:root:app.py: API-call GetHumidityValues
INFO:root:app.py: API-call GetHumidityValues
INFO:root:app.py: API-call ActivatePump
ERROR:app:Exception on /app/ActivatePump [GET]
Traceback (most recent call last):
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./app.py", line 56, in ActivatePump
    handler.ExecutePump()
  File "./Handler/Handler.py", line 55, in ExecutePump
    waterAmountCl = settings[0]["pump_water_amount"]
IndexError: list index out of range
INFO:root:app.py: API-call ActivatePump
INFO:root:app.py: API-call ActivatePump
INFO:root:app.py: API-call ActivatePump
ERROR:app:Exception on /app/ActivatePump [GET]
Traceback (most recent call last):
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./app.py", line 56, in ActivatePump
    handler.ExecutePump()
  File "./Handler/Handler.py", line 55, in ExecutePump
    waterAmountCl = settings[0]["pump_water_amount"]
IndexError: list index out of range
INFO:root:app.py: API-call ActivatePump
INFO:root:Handler.py: Executing Pump
INFO:root:app.py: API-call ActivatePump
INFO:root:Handler.py: Executing Pump
INFO:root:app.py: API-call ActivatePump
INFO:root:Handler.py: Executing Pump
INFO:root:app.py: API-call ActivatePump
INFO:root:Handler.py: Executing Pump
INFO:root:app.py: API-call ActivatePump
ERROR:app:Exception on /app/ActivatePump [GET]
Traceback (most recent call last):
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/pi/flask/appenv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./app.py", line 56, in ActivatePump
    handler.ExecutePump()
  File "./Handler/Handler.py", line 55, in ExecutePump
    waterAmountCl = settings[0]["pump_water_amount"]
IndexError: list index out of range
INFO:root:app.py: API-call GetSettings
INFO:root:app.py: API-call GetSettings
INFO:root:app.py: API-call GetSettings
INFO:root:app.py: API-call GetSettings
INFO:root:app.py: API-call GetSettings
INFO:root:app.py: API-call GetHumidityValues
